INSTALL.md

```diff
@@ -88,15 +87,19 @@ To prevent version conflicts, we recommend installing `scikit-learn-intelex` int
 - Install into a newly created environment (recommended):
 
   ```bash
-  conda create -n env -c conda-forge python=3.10 scikit-learn-intelex
+  conda config --add channels conda-forge
+  conda config --set channel_priority strict
```

Comment:
No, it doesn't work. it will installs package from default channel.  Year ago conda flow changed so now this actions are requeued.  This is process from conda-forge
https://conda-forge.org/

Alternatively users can use [Miniforge](https://github.com/conda-forge/miniforge/#download) instead of miniconda